### ISSUE TITLE
SET_GROUP:RemoveGroupsByName not working

### ISSUE THREAD

**Issue by dslawek** [FlightControl-Master/MOOSE#829]
**SET_GROUP:RemoveGroupsByName() not working.**

**Test script:**
```
function SetGroupAdd()
  SetGroup:AddGroupsByName( "TankGroup" )
  SetGroupFlush( "- after add TankGroup" )
end

function SetGroupRemove()
  SetGroup:RemoveGroupsByName( "TankGroup" )
  SetGroupFlush( "- after remove TankGroup" )
end

function SetGroupFlush( msg )
  env.info("SetGroup Flush START " .. msg )
  SetGroup:Flush()
  env.info("SetGroup Flush END   " .. msg )
end

SetGroup = SET_GROUP:New()
SetGroupFlush( "- Empty" )

SCHEDULER:New( nil, SetGroupAdd,    { }, 5 )
SCHEDULER:New( nil, SetGroupRemove, { }, 10, 10 )
```

**Log output:**
```
2018-03-16 13:27:47.904 INFO    SCRIPTING: SetGroup Flush START - Empty
2018-03-16 13:27:47.904 INFO    SCRIPTING:   5879(    13)/E:           SET_GROUP00045.Flush({[1]=Objects in Set:,[2]=,})
2018-03-16 13:27:47.904 INFO    SCRIPTING: SetGroup Flush END   - Empty
2018-03-16 13:27:53.002 INFO    SCRIPTING: SetGroup Flush START - after add TankGroup
2018-03-16 13:27:53.002 INFO    SCRIPTING:   5879(    13)/E:           SET_GROUP00045.Flush({[1]=Objects in Set:,[2]=TankGroup, ,})
2018-03-16 13:27:53.002 INFO    SCRIPTING: SetGroup Flush END   - after add TankGroup
2018-03-16 13:27:58.011 INFO    SCRIPTING: SetGroup Flush START - after remove TankGroup
2018-03-16 13:27:58.011 INFO    SCRIPTING:   5879(    13)/E:           SET_GROUP00045.Flush({[1]=Objects in Set:,[2]=TankGroup, ,})
2018-03-16 13:27:58.011 INFO    SCRIPTING: SetGroup Flush END   - after remove TankGroup
2018-03-16 13:28:08.019 INFO    SCRIPTING: SetGroup Flush START - after remove TankGroup
2018-03-16 13:28:08.020 INFO    SCRIPTING:   5879(    13)/E:           SET_GROUP00045.Flush({[1]=Objects in Set:,[2]=TankGroup, ,})
2018-03-16 13:28:08.020 INFO    SCRIPTING: SetGroup Flush END   - after remove TankGroup
2018-03-16 13:28:18.022 INFO    SCRIPTING: SetGroup Flush START - after remove TankGroup
2018-03-16 13:28:18.022 INFO    SCRIPTING:   5879(    13)/E:           SET_GROUP00045.Flush({[1]=Objects in Set:,[2]=TankGroup, ,})
2018-03-16 13:28:18.022 INFO    SCRIPTING: SetGroup Flush END   - after remove TankGroup
2018-03-16 13:28:28.039 INFO    SCRIPTING: SetGroup Flush START - after remove TankGroup
2018-03-16 13:28:28.039 INFO    SCRIPTING:   5879(    13)/E:           SET_GROUP00045.Flush({[1]=Objects in Set:,[2]=TankGroup, ,})
2018-03-16 13:28:28.039 INFO    SCRIPTING: SetGroup Flush END   - after remove TankGroup
```
**DCS:** DCS/2.5.0.15184 (x86_64; Windows NT 10.0.16299)
**MOOSE:** MOOSE GITHUB Commit Hash ID: 2018-03-15T20:38:41.0000000Z-559947f509519f008c2aae85dabae1a726af42f1

**Comment by dslawek** [comment]
No change - still not working...

**DCS:** DCS/2.5.2.18736
**MOOSE:** MOOSE GITHUB Commit Hash ID: 2018-06-19T21:12:15.0000000Z-8478268f5d857d6577c20c0c617c9bf56ec6aa68

**Comment by FlightControl-User** [comment]
sure? I need to test this in 2.4.

**Comment by FlightControl-User** [comment]
OK, fixed this and i can safely commit this fix in release 2.4, because the RemoveGroupsByName is not used anywhere within the MOOSE code. Fixed tested and closed.
